lib/ansible/modules/remote_management/ansible_playbook.py

```diff
@@ -0,0 +1,459 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2012, Red Hat, inc
```

Comment:
Did you write it 6 years ago?

Reply:
Nice catch @webknjaz. That's what happens with copy/paste.